apps/elewa-website/src/app/app.module.ts

```diff
@@ -4,13 +4,20 @@ import { HttpClientModule } from '@angular/common/http';
 
 import { AppComponent } from './app.component';
 import { TranslocoRootModule } from './transloco-root.module';
-
 import { AppRoutingModule } from './app.routing';
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [BrowserModule, AppRoutingModule],
-  providers: [HttpClientModule, TranslocoRootModule],
+  imports: [
+    BrowserModule, 
+    AppRoutingModule, 
+    ], 
+    providers: [
+      HttpClientModule,
+      TranslocoRootModule
+    ],
```

Comment:
The conflicts were resolved incorrectly. Kindly pull from main and refactor this.